fabtests/scripts/runfabtests.sh

```diff
@@ -130,12 +130,10 @@ functional_tests=(
 	"fi_recv_cancel -e rdm -V"
 	"fi_unexpected_msg -e msg -I 10"
 	"fi_unexpected_msg -e rdm -I 10"
-	"fi_inj_complete -e msg"
-	"fi_inj_complete -e rdm"
-	"fi_inj_complete -e dgram"
-	"fi_inj_complete -e msg -S"
-	"fi_inj_complete -e rdm -S"
-	"fi_inj_complete -e dgram -S"
+	"fi_msg_inject -A inject -v"
+	"fi_msg_inject -N -A inject -v"
+	"fi_msg_inject -A inj_complete -v"
+	"fi_msg_inject -N -A inj_complete -v"
 	"fi_bw -e rdm -v -T 1"
 	"fi_bw -e rdm -v -T 1 -U"
```

Comment:
Could you move this change to the beginning-ish of the list of commits (specifically to at least before you rename the test? This patch changes a lot of lines that were previously modified so putting it in the correct order will reduce the number of changes.
Also please move the -S -> -N into the patch that changes the flag